### PR TITLE
.NET SDK for Maintenance RP.

### PR DESCRIPTION
**Pull request by paul-selvaraj**
# Contributing to the Azure SDK

Please see our [CONTRIBUTING.md](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md) if you are not familiar with contributing to this repository or have questions.

For specific information about pull request etiquette and best practices, see [this section](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#pull-request-etiquette-and-best-practices).

### CODE REVIEW

**Review by archerzz — CHANGES_REQUESTED** [review]
@paul-selvaraj Make sure you update the following files, according to [the guide](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec).

- `AzSDK.RP.Props`
- `.csproj`
- `AssemblyInfo.cs`

File: sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Properties/AssemblyInfo.cs [hunk 1]
```diff
@@ -15,5 +15,5 @@
 [assembly: AssemblyCulture("")]
 
 
-[assembly: AssemblyVersion("1.4.0.0")]
-[assembly: AssemblyFileVersion("1.4.0.0")]
+[assembly: AssemblyVersion("1.5.0.0")]
+[assembly: AssemblyFileVersion("1.5.0.0")]
```
Comment (archerzz) [inline sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Properties/AssemblyInfo.cs:19]: You don't need to bump the `AsemblyVersion` if it's just a minor upgrade. See the [guide here](https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec).

> AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed

Comment (paul-selvaraj) [reply sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Properties/AssemblyInfo.cs]: Updated

**Review by archerzz — APPROVED** [review]

File: sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Properties/AssemblyInfo.cs (hunk 1 above)
Comment (archerzz) [reply sdk/maintenance/Microsoft.Azure.Management.Maintenance/src/Properties/AssemblyInfo.cs:19]: I've updated the versions. Actually for minor upgrade, bump `AssemblyFileVersion`, but don't bump `AssemblyVersion`. See https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/78/Mgmt-.NET-SDK-release-process?anchor=track-1-sdk-generation-from-updated-spec

> 6. Update the following information in AssemblyInfo.cs
      - AssemblyFileVersion (Must have)
      - AssemblyVersion: If it's a major version update (e.g. from 1.2.3.0 to 2.0.0.0), AssemblyVersion will also need to be changed

**Pull request merged**